<html><body><style>
			* {
				font-family:sans-serif;
			}
			body {
				text-align:center;
				padding:1em;
			}
			.messages {
				width:100%;
				max-width:700px;
				text-align:left;
				display:inline-block;
			}
			.messages img {
				background-color:rgb(248,244,240);
				width:36px;
				height:36px;
				border-radius:0.2em;
				display:inline-block;
				vertical-align:top;
				margin-right:0.65em;
			}
			.messages .time {
				display:inline-block;
				color:rgb(200,200,200);
				margin-left:0.5em;
			}
			.messages .username {
				display:inline-block;
				font-weight:600;
				line-height:1;
			}
			.messages .message {
				display:inline-block;
				vertical-align:top;
				line-height:1;
				width:calc(100% - 3em);
			}
			.messages .message .msg {
				line-height:1.5;
			}
		</style><div class="messages"><div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-15 18:21</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-15 18:30</div><div class="msg"><a href="https://www.frontiersin.org/articles/10.3389/fmicb.2017.01561/full">https://www.frontiersin.org/articles/10.3389/fmicb.2017.01561/full</a></div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-10-02/1397646232742_509e8f294a74b15e39a3_72.jpg" /><div class="message"><div class="username">kevens</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/403fe295956cd02c1b3513ae7f2107cf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png" /><div class="message"><div class="username">bohannan</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/053afcf45225777c5532ecb28060ff50.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0010-72.png" /><div class="message"><div class="username">ahickmot</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/4b990a58c1db2aecf972edaa9685f629.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0003-72.png" /><div class="message"><div class="username">dianachristie</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2022-03-13/3232606095026_bacbb29dba387631f658_72.png" /><div class="message"><div class="username">eyeh</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/d9009056777b15bcdd7893b704315fc2.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" /><div class="message"><div class="username">karadair</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/0ab1b6ad8b4331236cc017a046440561.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" /><div class="message"><div class="username">aonos</div><div class="time">2021-01-15 18:31</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-10-02/1397646232742_509e8f294a74b15e39a3_72.jpg" /><div class="message"><div class="username">kevens</div><div class="time">2021-01-15 19:05</div><div class="msg">set the channel description: The very best group for the very best kids.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/2413f240d07009d430d79c7584796169.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" /><div class="message"><div class="username">bethmillery</div><div class="time">2021-01-15 19:06</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/d9009056777b15bcdd7893b704315fc2.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" /><div class="message"><div class="username">karadair</div><div class="time">2021-01-15 19:45</div><div class="msg">Another paper that might be interesting to read:
<a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0237-y">https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0237-y</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-21 23:55</div><div class="msg">Reading the Hugerth paper for tomorrow reminded me of a paper from Susan Holmes with more specific advice on dimensionality reduction and visualization. It's a good paper to guide you any time you're putting together a PCA plot:
<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006907">https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006907</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/403fe295956cd02c1b3513ae7f2107cf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png" /><div class="message"><div class="username">bohannan</div><div class="time">2021-01-22 02:13</div><div class="msg">Thought it might be useful for us to have a google doc to collectively sketch out the basic microbiome analysis pipeline (and identify decision points that might be interesting to explore in more depth).  Here's a skeleton to begin with: <a href="https://docs.google.com/document/d/1ef3PryKEGvueUY1xlVuyGt8veJvqrDif1QQijWirCPI/edit?usp=sharing">https://docs.google.com/document/d/1ef3PryKEGvueUY1xlVuyGt8veJvqrDif1QQijWirCPI/edit?usp=sharing</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-22 19:39</div><div class="msg">For next week, we're going to talk about alpha diversity and normalization. One thing I think it's good to think about with normalization is why are we normalizing and for what analyses do you need to rarefy? For example, comparing alpha diversity between samples with different read counts would require some kind of normalization.

Here is the paper from Susan Holmes' group arguing against rarefying:
<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003531">https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003531</a>

Here is the paper Karen shared that compares different normalization strategies:
<a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0237-y">https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0237-y</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-22 19:52</div><div class="msg">Here's another paper that talks about alpha diversity and rarefaction together:
<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.02407/full">https://www.frontiersin.org/articles/10.3389/fmicb.2019.02407/full</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/403fe295956cd02c1b3513ae7f2107cf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png" /><div class="message"><div class="username">bohannan</div><div class="time">2021-01-27 22:43</div><div class="msg">Can someone remind us which paper we are reading for this week's lab meeting?  I remember that we are going to continue our discussion of alpha diversity from the Hugerth and Andersson paper...and I think one of the papers above?</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/d9009056777b15bcdd7893b704315fc2.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" /><div class="message"><div class="username">karadair</div><div class="time">2021-01-27 23:01</div><div class="msg">It might make sense to read the Willis paper (Frontiers) for the alpha diversity discussion?</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-27 23:04</div><div class="msg">I'm not sure what we decided on. We talked about reading the Waste Not Want Not paper as an argument against rarefying.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/d9009056777b15bcdd7893b704315fc2.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" /><div class="message"><div class="username">karadair</div><div class="time">2021-01-28 17:51</div><div class="msg">I'd like to discuss the Waste Not Want Not paper. Should we go with that one for this week?</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/403fe295956cd02c1b3513ae7f2107cf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png" /><div class="message"><div class="username">bohannan</div><div class="time">2021-01-28 22:02</div><div class="msg">Agreed!  Let's do that...</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-01-29 19:30</div><div class="msg">At the end of lab meeting today, we decided that next time we have a free lab meeting (Possibly February 19th?) we would discuss alpha diversity and beta diversity. Between now and then we should pick a paper to discuss. Possibly the Willis paper above unless there's a more recent or more thorough analysis of diversity metrics in microbiome research.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/d9009056777b15bcdd7893b704315fc2.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" /><div class="message"><div class="username">karadair</div><div class="time">2021-02-09 19:56</div><div class="msg">I didn't check this channel before suggesting Weiss et al. 2017. Open to other suggestions for a paper to discuss this Friday (2/12/21).</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-02-10 02:30</div><div class="msg">Weiss is fine with me. Maybe I'll present Willis or another follow up paper during my lab meeting next week.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-04-08 18:16</div><div class="msg">Hey, everyone.

Just wanted to share these resources for meta'omics studies. As I'm beginning to work with metagenomes, I've been looking for resources for both the theory and application of working with metagenomes. <a href="http://merenlab.org">http://merenlab.org</a> at the University of Chicago ran a meta'omics (metagenomes, metatranscriptomics, etc. ... not amplicon sequencing) seminar series over Zoom last summer and the videos are available on Youtube. <a href="https://merenlab.org/2020/06/27/seminar-series-on-microbial-omics/">https://merenlab.org/2020/06/27/seminar-series-on-microbial-omics/</a>.

In addition, they have created a software program for metagenomic analysis and visualization called Anvi'o, which I'm starting to learn. <a href="https://merenlab.org/software/anvio/">https://merenlab.org/software/anvio/</a>.

Not sure if anyone is planning on using meta'omics in their research, but I figured I'd share in case anyone is. Seems like good info on metagenomics is a lot fewer and farther between than on amplicon sequencing!</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-05-14 03:39</div><div class="msg"><a href="https://www.biorxiv.org/content/10.1101/2021.05.10.443486v1.full.pdf">https://www.biorxiv.org/content/10.1101/2021.05.10.443486v1.full.pdf</a>

Pretty interesting pre-print of a comparison of differential abundance test tools. They test pretty much all DA tools I've heard of including ones that require rarefied data and ones that don't. And they test them across 38 datasets (from various environments).

They find that tools that require rarefying and ones that don't both perform pretty much equally well in terms of identifying the same significant ASVs across tools and similar false-discovery rates. They also point out that whether or not you rarefy has a pretty minor effect compared to other data decisions (this is just for differential abundance, not for diversity metrics). They recommend filtering out ASVs that are only present in 10% of samples or fewer, because these low-prevalence taxa seriously hamper multiple testing corrections and thus lower the sensitivity of the tests. Also, variation in sequencing depth is correlated with the number of significant ASVs in pretty much every tool.

Overall, this paper concluded that ALDEx2 and ANCOM-II performed the best and were the most conservative (they had low false-discovery rates and identified a smaller number of taxa, but those taxa were identified by many other tools) and thus are also less sensitive. Another similar study identified different tools as the best: DESeq2 and corncob. So, I think any of those are valid choices. They also recommend running multiple of these tools in your analysis and then comparing the consensus taxa.

I usually use Corncob on non-rarefied data without filtering low-prevalence taxa. But I think in the future I will try ANCOM-II and start filtering taxa that are present in &lt;10% of samples.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2021-11-17 19:42</div><div class="msg">Hey, everyone. I updated my Talapas guide and added it to the lab website. Lemme know if there's anything you want to add to it or if there's something you want to do on Talapas that you don't know how to do:
<a href="https://bohannanlab.org/talapas/">https://bohannanlab.org/talapas/</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-02-11 19:30</div><div class="msg">As we talked about in lab meeting, it seems like the compositional data analysis (CoDA) methods are growing in popularity and are becoming the new standard. That being said, depending on your data, it may be that the "simpler" traditional methods, such as converting your counts to proportions and calculating Bray-Curtis dissimilarity, can provide similar conclusions. Though, your mileage may vary with what reviewers ask you to do.

If you want to start using compositional methods, a good place to start is the Aitchison dissimilarity metric, which is Euclidean distance after center-log ratio transformation (a.k.a, clr). The clr transformation takes the read count for a particular OTU in a particular sample and divides it by the geometric mean of all of the OTUs in that sample and log transforms that ratio. In this way, you can control for differences in library counts between samples while preserving the ratios between individual taxa. Do not perform any transformations on the read counts before clr - you want to use the raw reads.

To do this in R, I recommend using the `microViz` package function `dist_calc` or `tax_transform`. For example, if you want to plot an ordination you would do this:

```library(tidyverse)
library(phyloseq)
library(microViz)

phyloseq %&gt;%
    tax_transform(trans = "clr") %&gt;%
    ord_calc(method = "PCA") %&gt;%
    ord_plot()```
If you want to calculate Aitchison dissimilarities and use them in another analysis, such as a PERMANOVA, you would do this:
```phyloseq %&gt;%
    dist_calc(dist = "aitchison") %&gt;%
    dist_permanova()```
You don't necessarily have to use the `microViz` functions. You could just pull out the Aitchison dissimilarities and use them in `adonis2`  or manually create an ordination with `ggplot2` or whatever you're familiar with. The `transform` function from the `microbiome` package performs the actual clr transformation under the hood. If you're having trouble with any of this, feel free to ask me in Slack.

As a side note, if you want to use CoDa methods for differential abundance analysis, one option is `ALDEx2` . I haven't figured it out yet, but I intend to try it in future analyses.

Some good references:

Here is the original Aitchison citation:
Aitchison, J. (1982). The Statistical Analysis of Compositional Data. _Journal of the Royal Statistical Society: Series B (Methodological)_, _44_(2), 139?160. <a href="https://doi.org/10.1111/j.2517-6161.1982.tb01195.x">https://doi.org/10.1111/j.2517-6161.1982.tb01195.x</a>

This is a good review of compositional data analysis for microbiome data:
Calle, M. L. (2019). Statistical Analysis of Metagenomics Data. _Genomics &amp; Informatics_, _17_(1), e6. <a href="https://doi.org/10.5808/GI.2019.17.1.e6">https://doi.org/10.5808/GI.2019.17.1.e6</a>

Here is an extensive blog post about microbiome analysis using CoDA methods in R. It is currently my go-to reference for these types of analyses:
<a href="https://www.nicholas-ollberding.com/post/introduction-to-the-statistical-analysis-of-microbiome-data-in-r/">https://www.nicholas-ollberding.com/post/introduction-to-the-statistical-analysis-of-microbiome-data-in-r/</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/4b990a58c1db2aecf972edaa9685f629.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0003-72.png" /><div class="message"><div class="username">dianachristie</div><div class="time">2022-02-11 19:44</div><div class="msg">This is great thank you!!</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-03-04 18:56</div><div class="msg"><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z</a></div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2022-03-04/3208033396353_70999f8c7ac469d520fb_72.jpg" /><div class="message"><div class="username">robert.porch97</div><div class="time">2022-03-04 20:23</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-03-07 19:11</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-03-07 19:15</div><div class="msg">For those interested in down-sampling a dataset for permanova analysis, the code I used for this is here: <a href="https://github.com/maherrl/Aquarickettsia_bleaching_dynamics/blob/master/scripts/Down_sampling.R">https://github.com/maherrl/Aquarickettsia_bleaching_dynamics/blob/master/scripts/Down_sampling.R</a>
The repository also has the data I used for this script and a link to the paper where this analysis was published. Please reach out to me if you have any questions. I don't think anyone else has ever used this code so I'm not sure how easy it is to follow.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-03-07 19:19</div><div class="msg">I also want to add this link to bioinformatics lessons and tutorials which were invaluable to me in learning metagenomic and functional analyses. They also have a tutorial on analyzing 16S data with Qiime2, DaDa2, and phyloseq. <a href="https://github.com/biovcnet/biovcnet.github.io/wiki/TOPIC%3A-Amplicons">https://github.com/biovcnet/biovcnet.github.io/wiki/TOPIC%3A-Amplicons</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-03-11 20:01</div><div class="msg">Here is code on the Indicator Species Analysis. It may be a bit outdated. The repository readme also has the name of the paper so you can look up the corresponding figure.
<a href="https://github.com/maherrl/Poc_2016/blob/master/RLM_2016_Indicator_Species_Analysis_2018.R">https://github.com/maherrl/Poc_2016/blob/master/RLM_2016_Indicator_Species_Analysis_2018.R</a>

And here is code for a heat map of all samples. The corresponding paper with the figure is also linked in the readme.
<a href="https://github.com/maherrl/RAPID-analysis/blob/master/scripts/RAPID_ancom.R#L399">https://github.com/maherrl/RAPID-analysis/blob/master/scripts/RAPID_ancom.R#L399</a></div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-04-15 19:01</div><div class="msg">Here's the code I used for making venn diagrams for core taxa shared among different groups

```#install.packages("eulerr") # If not installed
library(eulerr)
library(microbiome)
#devtools::install_github('microsud/microbiomeutilities')
library(microbiomeutilities)
pig_states &lt;- unique(as.character(meta(pigment_rel)$pigment)) ### groups you are interested in phyoseq object
lineages &lt;- unique(as.character(meta(pigment_rel)$lineage))


for (n in pig_states){ # for each variable n in group
    #print(paste0("Identifying Core Taxa for ", n))
    
    pig.sub &lt;- subset_samples(pigment_rel, pigment == n) # Choose sample from group by n
    
    core_m &lt;- core_members(pig.sub, # ps.sub is phyloseq selected with only samples from g 
                           detection = 0.001, # 0.001 in atleast 75% samples 
                           prevalence = 0.75)
    print(paste0("No. of core taxa in ", n, " : ", length(core_m))) # print core taxa identified in each group.
    list_core_pig[[n]] &lt;- core_m # add to a list core taxa for each group.
    #print(list_core)
}


list_core_lin &lt;- c() # an empty object to store information

for (n in lineages){ # for each variable n in group
    #print(paste0("Identifying Core Taxa for ", n))
    
    pig.sub &lt;- subset_samples(pigment_rel, lineage == n) # Choose sample from DiseaseState by n
    
    core_m &lt;- core_members(pig.sub, # ps.sub is phyloseq selected with only samples from g 
                           detection = 0.001, # 0.001 in atleast 75% samples 
                           prevalence = 0.75)
    print(paste0("No. of core taxa in ", n, " : ", length(core_m))) # print core taxa identified in each DiseaseState.
    list_core_lin[[n]] &lt;- core_m # add to a list core taxa for each group.
    #print(list_core)
}

mycols &lt;- c(wildtype="#d6e2e9", mutant="#fcf5c7") 
plot(venn(list_core_pig),
     fills = mycols)

mycols &lt;- c(`Eel Creek`="#d6e2e9", `Rabbit Slough`="#cbf3f0", Mutant="#fcf5c7") 
plot(venn(list_core_lin),
     fills = mycols)```
</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-04-15 19:07</div><div class="msg">For making nice fixed effects plots from NBZIMM results:
```### convert results object to dataframe and merge with taxonomy table
res_zinb &lt;- get.fixed(f_zinb, part = "dist", vr.name = "Pigmentwildtype", sort = F)
res_zinb &lt;- as.data.frame(res_zinb)
res_filt_zinb &lt;- res_zinb %&gt;% filter(log(res_zinb$Std.Error) &lt; 5) ### I filtered out taxa that had extremely standard error for readability

res_filt_zinb_long &lt;- merge(na.omit(res_filt_zinb), tax, by = 0)
res_filt_zinb_long$confidence &lt;- NULL
res_filt_zinb_long$Row.names &lt;- as.character(res_filt_zinb_long$Row.names)

## get significant taxa for labeling:
res_sig_zinb &lt;- as.data.frame(res_zinb) %&gt;% filter(pvalue &lt; 0.05)
res_sig_zinb &lt;- merge(res_sig_zinb, tax, by = 0)

## ggplot! The labeling 

sig_zinb_taxa &lt;- paste(res_sig_zinb$Phylum, res_sig_zinb$Family, res_sig_zinb$Genus, sep = ";\n ")
res1_zinb_plot &lt;- res_filt_zinb_long %&gt;% ggplot(aes(y = Row.names, x = Estimate, xmin = Estimate-Std.Error, xmax = Estimate+Std.Error)) +
  theme_void() +
  geom_point(data = . %&gt;% filter(pvalue &gt; 0.05), color = "lightgray") +
  geom_errorbarh(color = "lightgray") +
  geom_point(size = 1.5, data = . %&gt;% filter(pvalue &lt; 0.05), aes(color = Row.names)) +
  geom_errorbarh(data = . %&gt;% filter(pvalue &lt; 0.05), aes(color = Row.names)) +
  ggrepel::geom_text_repel(data = . %&gt;% filter(pvalue &lt; 0.05), aes(label = pvalue), hjust = 0, size = 4) + 
  theme(axis.text.y = element_blank(),
        axis.title.y = element_blank(),
        axis.text.x = element_text(color = "black", size = 8, margin = margin(t = 5)),
        axis.title.x = element_text(color = "black", margin = margin(t = 10)),
        plot.background = element_rect(fill ="white", color = "white"),
        panel.background = element_rect(color = "gray"),
        panel.grid.major.x = element_line(colour = "gray"),
        legend.position = "left",
        legend.text=element_text(size=8)) +
  scale_color_manual(name = "Taxa", labels = sig_zinb_taxa) + ### this needs a vector for deciding what color pallete to use with "values = c()"
  guides(fill = guide_legend(label.position = "bottom",
                             title.position = "left", title.vjust = 1)) +
  ggtitle("Covariate: Pigment \nSignificant Taxa, p-value &lt; 0.05") +
  theme(legend.position = "none")```</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/d9009056777b15bcdd7893b704315fc2.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" /><div class="message"><div class="username">karadair</div><div class="time">2022-04-22 20:48</div><div class="msg"><a href="https://www.bioconductor.org/packages/devel/bioc/vignettes/ALDEx2/inst/doc/ALDEx2_vignette.html">https://www.bioconductor.org/packages/devel/bioc/vignettes/ALDEx2/inst/doc/ALDEx2_vignette.html</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/4b990a58c1db2aecf972edaa9685f629.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0003-72.png" /><div class="message"><div class="username">dianachristie</div><div class="time">2022-05-09 19:18</div><div class="msg"><a href="https://twitter.com/ElinVidevall/status/1523647268523753472?t=8MHxGGBnDWMiNbIe_oPkqw&amp;s=09">https://twitter.com/ElinVidevall/status/1523647268523753472?t=8MHxGGBnDWMiNbIe_oPkqw&amp;s=09</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-09 19:43</div><div class="msg">&gt; It's awesome that Silva tries to keep up with the nomenclature. Though worth considering whether that naming detail is worth losing 66% of the taxonomic assignments.
Oooh that's a hot take. :fire::fire::fire:

I'm surprised by that, but I've never tested it. I'll have to try this with my own data next time. If only there was a way to combine them and to use Silva's more up-to-date assignments and fall back on GG where Silva lacks an assignment.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/4b990a58c1db2aecf972edaa9685f629.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0003-72.png" /><div class="message"><div class="username">dianachristie</div><div class="time">2022-05-11 18:43</div><div class="msg">yea I was def not expecting this?</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-05-11 23:49</div><div class="msg">Did a quick comparison to look at # of NAs from using the different taxonomy classifiers on the stickleback dataset I'm working on</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-05-12 15:13</div><div class="msg">I wonder how they counted it. I did some poking around and Greengenes has a lot of rows that look like this in my data: "k__Bacteria; p__Chloroflexi; c__Chloroflexi; o__Herpetosiphonales; f__; g__; s__	1.0" where everything past order is unassigned, but is not counted as a NA in R when you separate the levels in cells. Silva doesn't format like that, so those cells are just filled with NAs.</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-10-02/1397646232742_509e8f294a74b15e39a3_72.jpg" /><div class="message"><div class="username">kevens</div><div class="time">2022-05-16 07:16</div><div class="msg">So is this to say you still got better (or more complete) results with Silva?</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-05-16 20:51</div><div class="msg">When I replace those "f__" etc cells with NAs, GG seems to do worse at IDing past Class and there are more NAs in the GG taxonomy tables than Silva. I am also curious how you can account for that in the confidence, since it might be easier to ID something to Order with higher confidence than down to Genus/Species. I think the original tweeter had a caveat that it could also depend on what environment you're sampling from.</div></div></div><br/>
<div><img src="https://avatars.slack-edge.com/2020-09-24/1398775480689_90affdf96ef747b7fb40_72.png" /><div class="message"><div class="username">csmith29</div><div class="time">2022-05-20 17:04</div><div class="msg"><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6#Sec9">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6#Sec9</a></div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-22 17:14</div><div class="msg">If you want to see Pat Schloss' hot takes on how to analyze microbiome data, check out his Youtube channel:
<a href="https://www.youtube.com/channel/UCGuktEl5InrcxPfCjmPWxsA">https://www.youtube.com/channel/UCGuktEl5InrcxPfCjmPWxsA</a>

Spoiler alert: he concludes rarefaction is better than any other approach.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 16:02</div><div class="msg">Hey, everyone. Just wanted to share something I've been working on.

Since we've been talking about different approaches to alpha and beta diversity, I wanted to compare some of these methods. So I took two 16S datasets, randomized them to remove the treatment effect while keeping the row and column sums, and then used different methods of calculating alpha and beta diversity. For alpha, I did observed richness (sobs), chao, average richness from 100 rarefactions (rare), and breakaway (est). For beta diversity, I did euclidean distance (norare), euclidean distance averaged from 100 rarefactions (rare), aitchison with rclr which ignores zeros, and aitchison with zclr which imputes values for the zeros.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 16:22</div><div class="msg">Wow what's going on with breakaway? I guess it makes sense that the other three alpha measures show essentially the same pattern. I would be curious to see a measure of evenness plotted for alpha diversity as well, since the ones included are basically all measures of richness.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 16:24</div><div class="msg">Where these soil samples? I would imagine that breakaway might predict higher richness for soil samples since we are much less likely to get a complete sampling of the community. Whereas for the coral microbiome (much lower diversity), I had similar plots in the past where breakaway and rare were the same.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 16:58</div><div class="msg">Interesting. Yeah, part of why I posted this was to see what it looks like in host systems. The "Brazil" plots are from tropical soils and the "AES" plots are from my soil bottle experiment.

It would make sense I guess if breakaway and rareified were more similar in a less complex microbiome. The AES experiment had much lower diversity and much deeper sequencing than the Brazil data and here both breakaway and aitchison look closer to rareified in the AES samples than Brazil, so I imagine as your sequencing depth increases relative to diversity they would agree more.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 17:10</div><div class="msg">This is the only thing I've done for corals. But this is divnet vs rarefied to ~1000 reads</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 17:14</div><div class="msg">Here is evenness.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 17:19</div><div class="msg">I think the argument for breakaway/divnet is that it is using a particular model of ratios between abundance classes to predict richness and therefore gets closer to the "true" diversity.

But I feel deeply uncomfortable reporting richness values based on that if they're so wildly variable. If your sample is diverse and your sequencing is shallow then fitting a model to incomplete data isn't gonna magically make your results better.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 17:21</div><div class="msg">Ya I think that is the worry...</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 17:23</div><div class="msg">From what I've heard from Amy, I don't think she ever imagined applying Divnet to samples with 20,000+ ASVs.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 17:26</div><div class="msg">So I guess which method you use depends on your question and system...and whether or not the dominant patterns change with the different methods. We have seen divnet estimates giving different patterns across treatments for corals at least, compared to rarefaction.

And I don't understand the beta diversity...I would have to think about those plots for a while haha</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 17:32</div><div class="msg">Maybe you already got this from looking at the plot... but just to explain it in case it's helpful. The y-axis is the pairwise dissimilarity between samples. The x-axis is the difference in sampling depth between the two samples being compared. So the first subplot shows that a greater difference in sequencing depth between two samples leads to a greater dissimilarity. So that's just an artifact of sequencing. Rarefaction and Aitchison both attempt to remove the effect of sequencing depth in different ways. It appears that rarefy generally does better at that than Aitchison and Aitchison is intermediate between rarefying and doing nothing.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 18:18</div><div class="msg">I guess I worry that the transformations remove some biologically relevant differences. I know you removed treatment, but would be interesting to see if any experimental or library prep information covaries with sequencing depth. For instance, bleached corals often have lower sequencing depth that unbleached corals, which would result in dissimilarity, but is that dissimilarity biologically relevant? I guess one what to do consider that is including sequencing depth as an offset, like the last model Caitlin showed us, but I'm not sure that correlates to multivariate analyses.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 18:51</div><div class="msg">I see what you're saying. For example, if treatment 1 has greater richness and greater sequencing depth than treatment 2, would that biological signal be removed if you apply one of these methods. In theory, if you pool equal amounts of DNA from each sample, sequencing depth should be independent of diversity (since most of your DNA will be PCR products). But that's not always true - for example in the AES dataset I have two treatments that have greater richness and greater sequencing depth than the other two. Also in theory, rarefying multiple times and taking the average richness should overcome the diversity loss from rarefying since you're repeatedly sampling from a larger metacommunity, but IDK if that would work with every dataset. I just did a quick and dirty comparison and rarefying doesn't seem to remove the biological signal in this dataset (this is the real data, not the randomized data):</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 18:55</div><div class="msg">Ok, good example. I think rarefaction is justified in this case. I would probably want to try it on another study without such a clear biological difference. But I guess it works!</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 18:56</div><div class="msg">Oh, and here is with the sequencing depth by treatment.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 18:59</div><div class="msg">Ya looks good.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 19:00</div><div class="msg">When I get my libraries sequenced I can try this with zebrafish guts.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 19:00</div><div class="msg">That'd be cool. Def share what you see.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 19:02</div><div class="msg">thank you for going down this rabbit hole with me, lol</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-05-24 19:04</div><div class="msg">Sure, my old labmate did a ton of work comparing patterns for corals with divnet vs rarefaction, so if you want to see those figures I can track them down. We went down this rabbit hole back then as well and realized we just needed to publish something haha</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-24 19:05</div><div class="msg">lol that is always the answer</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/62406881c6753acb1b8cfcb344000f50.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" /><div class="message"><div class="username">madeleine.gundersen</div><div class="time">2022-05-25 14:08</div><div class="msg">has joined #microbiome-analysis</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-05-29 19:10</div><div class="msg">I added the variance stabilizing transformation with a pseudocount of 1 from DESeq2 for the lulz.</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-06-21 23:21</div><div class="msg">Hey, microbial ecologists.

Does anyone know of a model or package to do an analysis of covariance (ANCOVA) with microbiome data? Essentially, I want to test the effect of a treatment on microbiome abundances after controlling for the abundances before the treatment was applied. Or is this done in other fields like expression analysis? For example, measuring gene expression or transcripts or something before and after an intervention?

Thanks!</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/f1842cbf41d2322c4571f19668d36290.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" /><div class="message"><div class="username">beccaluciamaher</div><div class="time">2022-06-21 23:43</div><div class="msg">Do you have before and after samples?</div></div></div><br/>
<div><img src="https://secure.gravatar.com/avatar/bebe7030c8340e3ce50e6e518971de05.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" /><div class="message"><div class="username">amorris3</div><div class="time">2022-06-22 00:55</div><div class="msg">Yes</div></div></div><br/>
</div></body></html>